docs/admin/addons.rst

```diff
@@ -184,8 +181,8 @@ The PO file header will contain list of contributors with years they have contri
 Update ALL_LINGUAS variable in the "configure" file
 ---------------------------------------------------
 
-Updates the ALL_LINGUAS variable in :file:`configure`, :file:`configure.in` or
-:file:`configure.ac` files, when a new translation is added.
+Updates the ALL_LINGUAS variable in :file:`configure`, :file:`configure.in` or the
+:file:`configure.ac` file, when a new translation is added.
```

Comment:
Are there more configure.ac files, or just the one?

Reply:
There can be one or more of the listed files in the repository.